openrtb_ext/bidders.go

```diff
@@ -247,6 +247,7 @@ var coreBidderNames []BidderName = []BidderName{
 	BidderZeroClickFraud,
 	BidderZetaGlobalSsp,
 	BidderZmaticoo,
+	BidderPixfuture,
```

Comment:
Move this elsewhere in the list so we maintain alphabetical order.

Reply:
The "BidderPixfuture," in the openrtb_ext/bidders.go have been moved to alphabetical order.